replica/table.cc

```diff
@@ -627,7 +627,9 @@ future<> storage_group_manager::for_each_storage_group_gently(std::function<futu
 
 void storage_group_manager::for_each_storage_group(std::function<void(size_t, storage_group&)> f) const {
     for (auto& [id, sg]: _storage_groups) {
```

Comment:
I think it should be documented that for_each_storage_group() skips closing groups.

Reply:
sure, will do it in a follow up since futurized variants also miss doc.